tiflash/tiflash-configuration.md

```diff
@@ -471,6 +471,13 @@ Note that the following parameters only take effect in TiFlash logs and TiFlash
 - This item is used for the MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. For more information, see [TiFlash MinTSO Scheduler](/tiflash/tiflash-mintso-scheduler.md).
 - Default value: `0`, which means twice the number of logical CPU cores
 
+##### `hashagg_use_magic_hash` <span class="version-mark">New in v9.0.0</span>
+
+- Controls the hash function used by TiFlash during aggregation operations. When set to `true`, TiFlash HashAgg uses the magic hash instead of the default CRC32.
```

Comment:
```suggestion
- Controls the hash function TiFlash uses for aggregation. When set to `true`, TiFlash HashAgg uses the magic hash instead of the default CRC32.
```